### PR TITLE
use qcli for qctools reports

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

**Comment by dericed** [comment]
could resolve https://github.com/amiaopensource/vrecord/issues/155

**Comment by privatezero** [comment]
Can't test right now, but looks good to me!

Would then have to update https://github.com/amiaopensource/homebrew-amiaos/blob/master/vrecord to require qcli then, right?

**Comment by dericed** [comment]
I'll make a release after the merge and then update the dependencies.

File: vrecord [hunk 1]
```diff
@@ -1193,8 +1193,12 @@ else
 fi
 
 if [[ "${qctoolsxml_choice}" = "Yes" ]] ; then
+    if [[ ! "$(which qclfi)" ]] ; then
```
Comment (kieranjol) [inline vrecord]: Is that f supposed to be there

**Comment by dericed** [comment]
Oops.

**Comment by retokromer** [comment]
What about the possibility to give to the user the chose between `qcli` (with an additional dependence) and `ffprobe` (without an additional dependence)?

**Comment by dericed** [comment]
updated for typo. to @retokromer both ffprobe and qcli are an additional dependence, this is the only place where ffprobe is used.

**Comment by retokromer** [comment]
@dericed As long as I remember, on brew ffprobe comes by default with ffmpeg.

**Pull request merged**